packages/components/src/checkbox-control/index.js

```diff
@@ -7,11 +7,25 @@ import { withInstanceId } from '@wordpress/compose';
  * Internal dependencies
  */
 import BaseControl from '../base-control';
+import { G, Path, SVG } from '../primitives';
 
 function CheckboxControl( { label, className, heading, checked, help, instanceId, onChange, ...props } ) {
 	const id = `inspector-checkbox-control-${ instanceId }`;
 	const onChangeValue = ( event ) => onChange( event.target.checked );
 
+	const svgCheck = (
+		<SVG
+			viewBox="0 0 20 20"
+			xmlns="http://www.w3.org/2000/svg"
+			className="components-checkbox-control__checked"
+			role="presentation"
+		>
+			<G>
+				<Path d="M14.83 4.89l1.34.94-5.81 8.38H9.02L5.78 9.67l1.34-1.25 2.57 2.4z" />
+			</G>
+		</SVG>
+	);
```

Comment:
Is this the same Dashicon available in the `Dashicon` component. Can we use the component instead to avoid loading the SVG twice in the bundle?

Reply:
It is, good call.